presto-native-execution/src/test/java/com/facebook/presto/hive/HiveExternalWorkerQueryRunner.java

```diff
@@ -174,7 +174,7 @@ public static QueryRunner createNativeQueryRunner(String baseDataDir, String pre
                         }
 
                         // Disable stack trace capturing as some queries (using TRY) generate a lot of exceptions.
-                        return new ProcessBuilder(prestoServerPath, "--logtostderr=1", "--v=1")
+                        return new ProcessBuilder(prestoServerPath, "--logtostderr=1", "--v=1", "--velox_exception_user_stacktrace=true")
```

Comment:
This used to cause problems with tests that generate lots of exceptions.

Reply:
> This used to cause problems with tests that generate lots of exceptions.

thanks for the reminder. but the issue is gone in the first of run after "--velox_exception_user_stacktrace=true" was added. I will try to rerun it a few times to see if i can get a stacktrace.